Development/nmos/server_utils.cpp

```diff
@@ -18,9 +20,26 @@ namespace nmos
 {
     namespace details
     {
+#if !defined(_WIN32) || defined(CPPREST_FORCE_HTTP_CLIENT_ASIO)
+        // This callback is called when client includes a certificate status request extension in the TLS handshake
+        static int server_certificate_status_request(SSL* s, void* arg)
+        {
+            nmos::experimental::ocsp_settings* settings = (nmos::experimental::ocsp_settings*)arg;
+            nmos::experimental::send_ocsp_response(s, nmos::with_read_lock(*settings->mutex, [&] { return settings->ocsp_response; }));
+            return SSL_TLSEXT_ERR_OK;
+        }
+
+        // setup server certificate status callback when client includes a certificate status request extension in the TLS handshake
+        void make_server_certificate_status_handler(boost::asio::ssl::context& ctx, const nmos::experimental::ocsp_settings& settings)
```

Comment:
I think this function, which returns `void` ought to be called something like `set_server_certificate_status_handler` (not `make`), didn't it? Also... just wondering... did you consider moving this function into _nmos/ocsp_utils.h_ so that the interface is at the level of `boost::asio::ssl::context`, similar to `http_listener_config`?